src/core/utils.js

```diff
@@ -822,3 +822,37 @@ export class InsensitiveStringSet extends Set {
         );
   }
 }
+
+export function makeSafeCopy(node) {
+  const clone = node.cloneNode(true);
+  clone.querySelectorAll("[id]").forEach(elem => elem.removeAttribute("id"));
+  clone.querySelectorAll("dfn").forEach(dfn => renameElement(dfn, "span"));
+  cleanupHyperComments(clone);
```

Comment:
Should we ultimately prevent comments everywhere? I'm not strongly convinced that only this position needs cleanup...

Reply:
Yeah, I’ve struggled with this too... I guess it would be ok to remove them all.